config/config_test.go

```diff
@@ -660,6 +661,7 @@ func TestFullConfig(t *testing.T) {
 	cmpInts(t, "experiment.adscert.inprocess.domain_renewal_interval_seconds", cfg.Experiment.AdCerts.InProcess.DNSRenewalIntervalInSeconds, 60)
 	cmpStrings(t, "experiment.adscert.remote.url", cfg.Experiment.AdCerts.Remote.Url, "")
 	cmpInts(t, "experiment.adscert.remote.signing_timeout_ms", cfg.Experiment.AdCerts.Remote.SigningTimeoutMs, 10)
+	cmpBools(t, "hooks.enabled", cfg.Hooks.Enabled, false)
```

Comment:
What I would like to see here, is for you to define the `hooks.enabled` variable in the `fullConfig` byte array that's defined on line 295, and set the variable equal to true, and assert that value here.

Reply:
Changed, thank you for the clarification.